build.sbt

```diff
@@ -1,67 +1,64 @@
 import sbt.Keys._
 import sbt._
-import uk.gov.hmrc.playcrosscompilation.PlayVersion.Play25
 
 val libName = "mongo-caching"
 
-val play25Version = "2.5.19"
-val play26Version = "2.6.23"
-val play27Version = "2.7.4"
+val play26Version = "2.6.25"
+val play27Version = "2.7.9"
+val play28Version = "2.8.7"
 
 val compileDependencies = PlayCrossCompilation.dependencies(
   shared = Seq(
     "uk.gov.hmrc"       %% "time"               % "3.8.0"
   ),
-  play25 = Seq(
-    "com.typesafe.play" %% "play"                 % play25Version    % Provided,
-    "uk.gov.hmrc"       %% "simple-reactivemongo" % "7.30.0-play-25",
-    "uk.gov.hmrc"       %% "http-verbs"           % "10.6.0-play-25"
-  ),
   play26 = Seq(
-    "com.typesafe.play" %% "play"                 % play26Version    % Provided,
-    "uk.gov.hmrc"       %% "simple-reactivemongo" % "7.30.0-play-26",
-    "uk.gov.hmrc"       %% "http-verbs"           % "10.6.0-play-26"
+    "com.typesafe.play" %% "play"                 % play26Version,
```

Comment:
no longer `Provided` - so normal eviction applies (excludes required if need to downgrade) which makes compatibilities more debuggable (e.g. `dependencyTree`)